docs/Makefile

```diff
@@ -7,6 +7,10 @@ SPHINXBUILD   = sphinx-build
 SPHINXPROJ    = ConcepttoClinic
 SOURCEDIR     = .
 BUILDDIR      = _build
+APIDOCDIR_INTERFACE    = _apidoc_interface
+APIDOCDIR_PREDICTION   = _apidoc_prediction
```

Comment:
Multiple apidoc directories are necessary since for each package a 'modules.rst' is created, which would be overwritten by the last execution of `sphinx-apidoc` when the output should only be saved in one directory.